tests/test_scrapy_spider.py

```diff
@@ -1,4 +1,5 @@
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import
```

Comment:
2 options for this test:

- regarding `sgmllib` not available in Python 3, I suggest you add a _skip in Python 3_ for  `test_scrapy_spider`
- or remove `RegexLinkExtractor` from https://github.com/scrapinghub/frontera/blob/master/tests/scrapy_spider/spiders/example.py#L23